### PR TITLE
[New community extension] UpdateChecker

### PR DESCRIPTION
**Pull request by arthuro555**
An extension to check if the version of the game on liluo is different to the current game version. This allows for a basic update check for PC and mobile builds of games. Thanks arthuro555!

### CODE REVIEW

**Comment by 4ian** [comment]
👍

File: extensions/community/UpdateChecker.json [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+      "objectGroups": []
+    },
+    {
+      "description": "Checks if the game version on liluo is different from the one of the currently running game. It may take a few seconds for the liluo game version to be obtained, and in this time frame, the condition will not trigger. If disconnected from the internet, this will not function.",
+      "fullName": "Update is available",
+      "functionType": "Condition",
+      "group": "",
+      "name": "UpdateAvailable",
+      "private": false,
+      "sentence": "An update is is available",
```
Comment (Bouh) [inline extensions/community/UpdateChecker.json:111]: `is is`

File: extensions/community/UpdateChecker.json [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+              ]
+            }
+          ]
+        }
+      ],
+      "parameters": [],
+      "objectGroups": []
+    },
+    {
+      "description": "Loads the latest game version from liluo. This is already called at the beginnign of the scene, but you can still re-call it if it failed or if you want to add a \"check again\" button. Make sure to not call this too often (or at all), as otherwise it will DDoS the version-checking server!.",
```
Comment (Bouh) [inline extensions/community/UpdateChecker.json:51]: `beginnign`

File: extensions/community/UpdateChecker.json [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+                "__UpdateChecker.Error",
+                ""
+              ]
+            },
+            {
+              "type": {
+                "value": "SendAsyncRequest"
+              },
+              "parameters": [
+                "\"https://updates.arthuro555.com/?game-id=\" + UpdateChecker::CurrentGameID()",
```
Comment (Bouh) [inline extensions/community/UpdateChecker.json:90]: Can you explain this request to your server, please?

Also, a request on a server always raises the question of its maintenance and availability.

File: extensions/community/UpdateChecker.json [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+          "parameterObjects": "",
+          "useStrict": true,
+          "eventsSheetExpanded": false
+        }
+      ],
+      "parameters": [],
+      "objectGroups": []
+    },
+    {
+      "description": "Gets the game version from the project properties",
```
Comment (Bouh) [inline extensions/community/UpdateChecker.json:211]: Missing dot.

File: extensions/community/UpdateChecker.json [hunk 5]
```diff
@@ -0,0 +1,273 @@
...
+              ]
+            }
+          ]
+        }
+      ],
+      "parameters": [],
+      "objectGroups": []
+    },
+    {
+      "description": "Checks if the game version on liluo is different from the one of the currently running game. It may take a few seconds for the liluo game version to be obtained, and in this time frame, the condition will not trigger. If disconnected from the internet, this will not function.",
```
Comment (Bouh) [inline extensions/community/UpdateChecker.json:105]: `Liluo` with big caps, like you wrote in the other descriptions.

File: extensions/community/UpdateChecker.json (hunk 2 above)
Comment (Bouh) [inline extensions/community/UpdateChecker.json:51]: `Liluo`

File: extensions/community/UpdateChecker.json [hunk 6]
```diff
@@ -0,0 +1,273 @@
...
+  "fullName": "Update checker",
+  "helpPath": "",
+  "iconUrl": "data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz48IURPQ1RZUEUgc3ZnIFBVQkxJQyAiLS8vVzNDLy9EVEQgU1ZHIDEuMS8vRU4iICJodHRwOi8vd3d3LnczLm9yZy9HcmFwaGljcy9TVkcvMS4xL0RURC9zdmcxMS5kdGQiPjxzdmcgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgdmVyc2lvbj0iMS4xIiBpZD0ibWRpLXVwZGF0ZSIgd2lkdGg9IjI0IiBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiPjxwYXRoIGQ9Ik0yMSwxMC4xMkgxNC4yMkwxNi45Niw3LjNDMTQuMjMsNC42IDkuODEsNC41IDcuMDgsNy4yQzQuMzUsOS45MSA0LjM1LDE0LjI4IDcuMDgsMTdDOS44MSwxOS43IDE0LjIzLDE5LjcgMTYuOTYsMTdDMTguMzIsMTUuNjUgMTksMTQuMDggMTksMTIuMUgyMUMyMSwxNC4wOCAyMC4xMiwxNi42NSAxOC4zNiwxOC4zOUMxNC44NSwyMS44NyA5LjE1LDIxLjg3IDUuNjQsMTguMzlDMi4xNCwxNC45MiAyLjExLDkuMjggNS42Miw1LjgxQzkuMTMsMi4zNCAxNC43NiwyLjM0IDE4LjI3LDUuODFMMjEsM1YxMC4xMk0xMi41LDhWMTIuMjVMMTYsMTQuMzNMMTUuMjgsMTUuNTRMMTEsMTNWOEgxMi41WiIgLz48L3N2Zz4=",
+  "name": "UpdateChecker",
+  "previewIconUrl": "https://resources.gdevelop-app.com/assets/Icons/update.svg",
+  "shortDescription": "Checks if the version of the game on liluo is different than the current game version.",
```
Comment (Bouh) [inline extensions/community/UpdateChecker.json:11]: `Liluo`

**Pull request merged**